cockatrice/src/tab_game.cpp

```diff
@@ -802,6 +802,27 @@ Player *TabGame::addPlayer(int playerId, const ServerInfo_User &info)
     gameMenu->insertMenu(playersSeparator, newPlayer->getPlayerMenu());
 
     players.insert(playerId, newPlayer);
+
+    int i = 1;
```

Comment:
declare in for statement

Reply:
I use it a few lines below for this `newPlayer->setZoneId(i);`, maybe should I give the variable a more  significant name?

Reply:
no you should move that inside the loop